src/static/redirects.njk

```diff
@@ -17,11 +17,28 @@ eleventyExcludeFromCollections: true
 # Internal Redirects
 /demo/*                             /play/:splat 302!
 
-# Proxied Endpoints
-/docs/*                             https://docs-eslint.netlify.app/:splat 200!
+{% if site.language.code == "en" %}
+
+# Old-Style Docs to New-Style Docs
+/docs/rules/*                       /docs/latest/rules/:splat 301!
+/docs/user-guide/*                  /docs/latest/user-guide/:splat 301!
+/docs/maintainer-guide/*            /docs/latest/maintainer-guide/:splat 301!
+/docs/developer-guide/*             /docs/latest/developer-guide/:splat 301!
```

Comment:
This seems redundant if we'll have `/docs/*                             /docs/latest/:splat 301!` few lines below?

Reply:
Oops! Meant to remove that. Thanks!